rails/app/views/welcome/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="welcome">
+<div class="welcome" style="background-image: url(<%= @theme ? @theme.background_img : 'welcome-bg.jpg' %>) " > <%#no-repeat center center fixed%>

```

Comment:
I noticed that when there is no theme set, the default welcome image is missing / a blank white background.

I believe that's because now that we are referencing the image from the root (rather than relative assets in the stylesheet), we need to specify the assets folder.

```suggestion
<div class="welcome" style="background-image: url(<%= @theme ? @theme.background_img : 'assets/welcome-bg.jpg' %>) " >
```